test/spec/unit/bootstraps/liveblogTest.js

```diff
@@ -140,7 +140,6 @@ define([
             it('sets up scroll event listener', function () {
                 liveblog.init();
 
-                expect(window.addEventListener).to.have.been.calledOnce;
```

Comment:
This is fine. Could we add creativeInjector to the mocked dependencies in this test spec? Otherwise when the liveblog tests are run they'll import creativeInjector and go inside this module.